src/app/examples/empty-state-example/empty-state-example.component.tns.html

```diff
@@ -0,0 +1,3 @@
+<kirby-empty-state [iconName]="iconName" [title]="title" [text]="text">
```

Comment:
Consider inlining these variables, as it will make the example code more clear

Reply:
Fixed